louiseA/file-1.js

```diff
@@ -0,0 +1,103 @@
+1.
+
+ a) "1" == 1
+  true
+
+ b) "1" === 1
+  false
+
+ c) x == "x"
+  true
+
+ d) x == (x+'')
+  false
+
+ e) '' == ' '
+  true
+
+ f) x = true
+  true
+
+ g) var x; x == 'undefined'
+  false
+
+ h) '9' > '10'
+  true
+
+ i) typeof x + 1 === 'number'
+  false
+
+ j) typeof x % 2 === 'number'
+  false
+
+ k) typeof (x % 2) === 'number'
+  true
+
+ l) x++ == ++x //having trouble understanding the increment operator...
+
+ m) ++x == x++ //see above
+
+ n) "1"+x == 1+x
+  false, unless x is a string
+
+ o) "0" +1 == 1
+ true
+
+ p) (typeof (x+1)) === (typeof x)
+  true, if x is a number
+
+ q) (x*1 == x) || ((typeof x) != "number")
+  true, if x is a number
+
+ r) (x=(typeof (x+(typeof x)))) == x
+  false
+
+
+2.
+
+ a) x+y+z/3
+
+ b) (x/2)+(y/2)+(z/2)
+
+3.
+
+  a) (l+r)(t+b)
+
+  b) (t,b) > (l+r)
+
+  c) /*I know the rules for finding circumference/area of circles, but
+  I'm struggling with getting the variables right if that makes sense */
+
+  d) //see above
+
+  e) //see above
+
+4.
+
+ a) /*I feel like there may be something I'm missing here because I came up with
+ an exression that worked for the '7' column, but when I tried it elsewhere it
+ wouldn't work and I had to adjust the patterns and I didn't solve what was
+ going on but here's the expressions I did come up with... */
+  7s column:
+  ((R+1)C ) + R))
+  5s column:
+  (((R+1)C)=x)+(x/2)
+
+ b) //see explanation above
+
+ c) //see explanation  above
+
+ d) var color;
+  if N= isEven {
+   color= black;
+   } else {
+   color = white;
+   }
+
+5. /* unfortunately I left these problems for last, and not only did I immediately
+think "WTF is Math.floor" and not google it, but I had difficulty with putting
+the fraction translation into string form */
+
+a)
+
+b)
```

Comment:
Hey Louise, looks like you understand what's going on in the problems, but just needed some help being pointed in the right direction. If you want explanations on the solutions, I'd be more than happy to help out during office hours, and I'm sure the other TA's would as well! Also, trying to find the time to code with other people, and asking a lot of questions, is also really helpful! This stuff is confusing at first, but the more you keep at it, the easier it is to understand :)